database/sql/V3__IBAN.sql

```diff
@@ -0,0 +1,112 @@
+       nullif(det.CHIAVE_AVVIO, '')    CHIAVE_AVVIO,
+       nullif(det.CHIAVE_ESITO, '')    CHIAVE_ESITO,
+       det.OBJ_ID                      OBJ_ID,
+       mas.OBJ_ID                      MASTER_OBJ
+from INFORMATIVE_CONTO_ACCREDITO_MASTER mas
+         inner join (
+    select max(OBJ_ID) OBJ_ID
+    FROM INFORMATIVE_CONTO_ACCREDITO_MASTER
+    where DATA_INIZIO_VALIDITA <= CURRENT_TIMESTAMP
+    GROUP BY FK_PA
+    HAVING max(DATA_INIZIO_VALIDITA) <= CURRENT_TIMESTAMP
+) right_id_by_pk
+                    on mas.OBJ_ID = right_id_by_pk.OBJ_ID
+         inner join INFORMATIVE_CONTO_ACCREDITO_DETAIL det
+                    on mas.OBJ_ID = det.FK_INFORMATIVA_CONTO_ACCREDITO_MASTER;
+
+-- Insert data
+
+INSERT INTO NODO4_CFG.CODIFICHE (OBJ_ID, ID_CODIFICA, DESCRIZIONE) VALUES (1, 'BARCODE-GS1-128', 'Codifica barcode GS1-128 tot 13');
+INSERT INTO NODO4_CFG.CODIFICHE (OBJ_ID, ID_CODIFICA, DESCRIZIONE) VALUES (2, 'QR-CODE', 'QR-CODE CF  11 caratteri');
+INSERT INTO NODO4_CFG.CODIFICHE (OBJ_ID, ID_CODIFICA, DESCRIZIONE) VALUES (3, 'BARCODE-128-AIM', 'BARCODE-128-AIM zeri + ccp tot 12');
+
+
+INSERT INTO NODO4_CFG.CODIFICHE_PA (OBJ_ID, CODICE_PA, FK_CODIFICA, FK_PA) VALUES (219, '00168480242', 2, 190);
+INSERT INTO NODO4_CFG.CODIFICHE_PA (OBJ_ID, CODICE_PA, FK_CODIFICA, FK_PA) VALUES (220, '8088888169189', 1, 190);
+
+INSERT INTO NODO4_CFG.BINARY_FILE (OBJ_ID, FILE_CONTENT, FILE_HASH, FILE_SIZE, SIGNATURE_TYPE, XML_FILE_CONTENT) VALUES (201, '3C3F786D6C2076657273696F6E3D22312E302220656E636F64696E673D225554462D38223F3E0D0A3C696E666F726D6174697661436F6E746F41636372656469746F20786D6C6E733A7873693D22687474703A2F2F7777772E77332E6F72672F323030312F584D4C536368656D612D696E7374616E636522207873693A6E6F4E616D657370616365536368656D614C6F636174696F6E3D22496E666F726D6174697661436F6E746F41636372656469746F5F315F325F302E787364223E0D0A093C6964656E7469666963617469766F466C7573736F3E435F413730332D4942414E2D32303137303330383030303030303C2F6964656E7469666963617469766F466C7573736F3E0D0A093C6964656E7469666963617469766F446F6D696E696F3E30303136383438303234323C2F6964656E7469666963617469766F446F6D696E696F3E0D0A093C726167696F6E65536F6369616C653E436F6D756E652064692042617373616E6F2064656C204772617070613C2F726167696F6E65536F6369616C653E0D0A093C64617461507562626C6963617A696F6E653E323031372D30332D30385430303A30303A30303C2F64617461507562626C6963617A696F6E653E0D0A093C64617461496E697A696F56616C69646974613E323031372D30332D30395430303A30303A30303C2F64617461496E697A696F56616C69646974613E0D0A093C636F6E7469446941636372656469746F3E0D0A09093C212D2D20636F6F7264696E6174652062616E63617269652064656C20636F6E746F20636F7272656E7465207072696E636970616C65206469207465736F7265726961202D2D3E0D0A09093C6962616E41636372656469746F3E495435304D303230303836303136353030303030333439373438313C2F6962616E41636372656469746F3E0D0A093C2F636F6E7469446941636372656469746F3E0D0A3C2F696E666F726D6174697661436F6E746F41636372656469746F3E0D0A', '4B57E35E75CEDADAEF5CF4CCA0D6CF8BE9965725', 724, null, null);
```

Comment:
It is needed because is referenced by INFORMATIVE_CONTO_ACCREDITO_MASTER. The file contains the info about iban in xml (@corvettafabio). In the future we will develop the download of this file